packages/react-ui-rent/src/LeadForm/Fields/OptInNewsLetter.js

```diff
@@ -1,27 +1,57 @@
 import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
+import themed from 'react-themed'
 import { Field } from '@rentpath/react-ui-core'
 
-export default class Email extends PureComponent {
+@themed(/^OptInNewsLetter/, {
+  pure: true,
+})
+
+export default class OptInNewsLetter extends PureComponent {
   static propTypes = {
+    theme: PropTypes.object,
     name: PropTypes.string,
+    id: PropTypes.string,
     type: PropTypes.string,
     label: PropTypes.oneOfType([
       PropTypes.string,
       PropTypes.object,
       PropTypes.func,
     ]),
+    defaultChecked: PropTypes.bool,
   }
 
   static defaultProps = {
+    theme: {},
     name: 'opt_in_newsletter',
+    id: 'lead_form_opt_in_newsletter',
```

Comment:
If I have multiple lead forms on the page with an `OptInNewsLetter`, how does distinguish between the two? I guess I'm not understanding what the purpose of this `id` is here.

Reply:
If you're trying to use this for `htmlFor`, then please take a look at https://github.com/rentpath/react-ui/blob/master/packages/react-ui-utils/src/randomId.js.  You can look in react-ui-core because it has been used there, especially in regards to `Field`.